### PR TITLE
Community Spotlight Update [April] [2024]

### PR DESCRIPTION
**Pull request by taniashiba**
**Updated Community Spotlight for last month, April 2024.**

Members of the month:

- fussofbug5 - Rising Star
- abyteham - Rising Star
- Josephweb3 - Community Champion
- planetnineisaspaceship - Feedback Champion
- liru8394 - Community Champion
- sondreb - Trend Setter

### CODE REVIEW

**Review by blackgirlbytes — APPROVED** [review]
i love the trendsetter award!!!

File: site/src/contributors.json [hunk 1]
```diff
@@ -1,39 +1,39 @@
...
+    "description": "You consistently ask great questions, are working hard to learn Web5 and help other community members get started just by posting. Thank you for being here! ❤️"
   },
   {
-    "login": "rbe",
-    "avatar_url": "https://cdn.discordapp.com/avatars/114142818460631046/578c2b6bc2d97553021b459478794786?size=1024",
-    "contributions": ["Feedback Champion Award"],
-    "description": "You have been consistently supportive and talkative about all things TBD, and you have given our team great feedback already. Thank you so much for being part of our community! 🥰"
+    "login": "sondreb",
+    "avatar_url": "https://cdn.discordapp.com/avatars/185450368782696448/b8531ccdd4257d2091ca4cae6bd6150b?size=1024",
+    "contributions": ["Trend Setter Award"],
```
Comment (blackgirlbytes) [inline site/src/contributors.json:35]: hmmmm I could be wilding but I think Trendsetter is one word 🤔

**Review by blackgirlbytes — CHANGES_REQUESTED** [review]

File: site/src/contributors.json (hunk 1 above)
Comment (taniashiba) [reply site/src/contributors.json:35]: Oh it is! I went with spelling of previous word o:

Comment (taniashiba) [reply site/src/contributors.json]: Addressed with new changes!

**Review by blackgirlbytes — APPROVED** [review]

**Comment by taniashiba** [comment]
Thank yall 🥺 🙏 🖤

**Pull request merged**